librosa/filters.py

```diff
@@ -599,7 +599,7 @@ def constant_q(sr, fmin=None, n_bins=84, bins_per_octave=12, tuning=0.0,
     if pad_fft:
         max_len = int(2.0**(np.ceil(np.log2(max_len))))
     else:
-        max_len = np.ceil(max_len)
+        max_len = int(np.ceil(max_len))
```

Comment:
> since the first one might cause unexpected results if it doesn't assume the size to be only int-valued.

I think this is fine, since the docstring for `pad_center` explicitly states that `size` should be an integer type.



> Can I ask you what do you think how to fix this in util.pad_center?

I would do just as you suggest: ``size = int(size)``.  I wouldn't worry too much about being bullet-proof here, since the `util` submodule is primarily for use by other parts of librosa.  As long as those parts do the right thing, and the behavior is well-documented, I'm happy.